src/components/statsBar/StatsView.js

```diff
@@ -66,41 +66,42 @@ const styles = () => ({
     margin: '0 auto',
   }),
   statTitle: (props) => ({
-    float: 'left',
-    color: '#062D4F',
-    fontFamily: 'Nunito',
-    fontWeight: 'bold',
-    fontSize: '11px',
+    float: props.statsStyling.statTitle ? props.statsStyling.statTitle.float ? props.statsStyling.statTitle.float : 'left' : 'left',
```

Comment:
something doesn't seem right multiple ternaries? Is there any better way to do this?
Can you add a sample story for this so can verify this

Reply:
@AjayDoddapaneni  Added a sample stub 'icdcstatsBar'